### PR TITLE
Split existing Scenarios in two : Simulations and Scenarios

### PR DESCRIPTION
**Pull request by mbaechler**
- Scenarios being building blocks are now located in main
	- Simulation are still in test because it's what gatling expects
	- Some reorganization of packages to be consistent with the split

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]

**Review by blackheaven — APPROVED** [review]

File: src/it/scala-2.12/org/apache/james/gatling/JmapIT.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.apache.james.gatling.{Fixture, JamesServer}
+import org.slf4j
+import org.slf4j.LoggerFactory
+
+abstract class JmapIT extends GatlingFunSpec {
+  val logger: slf4j.Logger = LoggerFactory.getLogger(this.getClass.getCanonicalName)
+
+  private val server: RunningServer = JamesServer.start()
+  lazy val protocolConf: Protocol = http.baseUrl(s"http://localhost:${server.mappedJmapPort}")
+  before(server.addUser(bart))
```
Comment (rouazana) [inline src/it/scala-2.12/org/apache/james/gatling/JmapIT.scala:22]: D'oh! Where is Homer?

**Review by rouazana — APPROVED** [review]
nice, did you check if you fixed https://github.com/linagora/james-gatling/issues/54 ?

**Review by aduprat — APPROVED** [review]

**Comment by mbaechler** [comment]
> nice, did you check if you fixed #54 ?

no I didn't. We can prioritize that ticket in next sprint if you want.

**Comment by aduprat** [comment]
Already merged

**Pull request closed without merging**